main.go

```diff
@@ -354,14 +354,14 @@ func resultPostProcess(hasChange bool, deprecatedMessagesCh chan string, originF
 			return
 		}
 
-		if err := os.WriteFile(originFilePath, formattedOutput, 0644); err != nil {
+		if err := os.WriteFile(originFilePath, formattedOutput, 0o644); err != nil {
```

Comment:
what is 0o644? correct is 0644

Reply:
`0o644` appeared in Go 1.13. In 2023, it's the preferred way to write octal literals.

I provided links in the PR's description:
> An integer literal is a sequence of digits representing an [integer constant](https://go.dev/ref/spec#Constants). An optional prefix sets a non-decimal base: 0b or 0B for binary, 0, 0o, or 0O for octal, and 0x or 0X for hexadecimal. A single 0 is considered a decimal zero. In hexadecimal literals, letters a through f and A through F represent values 10 through 15.

Also, from the https://go.dev/doc/go1.13:
> [Octal integer literals](https://go.dev/ref/spec#Integer_literals): The prefix 0o or 0O indicates an octal integer literal such as 0o660. The existing octal notation indicated by a leading 0 followed by octal digits remains valid.

More info in the original proposal: https://github.com/golang/go/issues/12711